cmd/scheduler/options/options.go

```diff
@@ -83,6 +84,7 @@ func New(origArgs []string) *Options {
 	fs.StringSliceVar(&opts.EtcdInitialPeers, "initial-cluster", []string{"dapr-scheduler-server-0=http://localhost:2380"}, "Initial etcd cluster peers")
 	fs.StringVar(&opts.EtcdDataDir, "etcd-data-dir", "./data", "Directory to store scheduler etcd data")
 	fs.StringSliceVar(&opts.EtcdClientPorts, "etcd-client-ports", []string{"dapr-scheduler-server-0=2379"}, "Ports for etcd client communication")
+	fs.StringSliceVar(&opts.EtcdClientHttpPorts, "etcd-client-http-ports", []string{""}, "Ports for etcd client http communication")
```

Comment:
Should we put a sane default here? or no and default is non production setup

Reply:
Thanks for the review Cassie 🙏 
I don't know, my feeling is that we should stay as close as possible to etcd behaviour here, and keep it empty, so that we have http and grpc on the same port. This lowers the complexity for development, while still keeping open the possibility of adding a separate http port for production.